src/renderer/util.ts

```diff
@@ -24,7 +24,7 @@ export const extractFileExtension: (filePath: string) => string | null = (
 export const getMediaType: (extension: string) => 'audio' | 'video' | null = (
   extension
 ) => {
-  const audioFileExtensions = ['mp3'];
+  const audioFileExtensions = ['mp3', 'mpeg'];
```

Comment:
`.mpeg` is a video file format, it is not? https://www.howtogeek.com/367560/what-is-an-mpeg-file-and-how-do-i-open-one/#:~:text=A%20file%20with%20the%20.,than%20other%20popular%20video%20formats.

To be fair I don't think I've ever seen a file with this extension, any reason we want to support it?

Reply:
because thats how the drop handler detects audio files - not as mp3 which the file is, it just categories it all as mpeg - http://www.differencebetween.net/technology/protocols-formats/difference-between-mpeg-and-mp3/#:~:text=Actually%2C%20an%20MP3%20is%20just,compression%20of%20the%20audio%20component. According to this mp3 is a subset of mpeg

Reply:
<img width="442" alt="bugfix drag" src="https://user-images.githubusercontent.com/55219708/184558325-d1909398-04e4-4b1c-b213-8ee29ad0c9fe.PNG">

Reply:
@chloebrett  see new logic - narrow it further if that helps